src/ontology/lexmatch/mondo-only/unmapped_gard_mondo.tsv

```diff
@@ -0,0 +1,2 @@
+subject_id	subject_label	object_id	predicate_id	object_label	mapping_justification	mapping_tool	confidence	subject_match_field	object_match_field	match_string
+ID		A oboInOwl:hasDbXref	>A oboInOwl:source							
```

Comment:
@hrshdhgd I don't know much about lexmatch. I just assume it proposes mappings based on string similarity. Is that correct?

What is it that decides what will go into `unmapped_gard_mondo.tsv` and `unmapped_gard_mondo_exact.tsv`? I realize these are empty right now, and I'm guessing that is expected, but I know that there are many unmapped terms, including ones that will [still be unmapped](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=701107044) after I shortly finished with the proxy logic.

Reply:
`unmapped_gard_mondo.tsv` will have all predicates and `unmapped_gard_mondo_exact.tsv` has on `skos:exactMatch`